froi/core/dataobject.py

```diff
@@ -786,14 +786,40 @@ def __init__(self, surf_path, offset=None):
             applied. If != 0.0, an additional offset will be used.
 
         """
-        self.surf = SurfaceDataset(surf_path, offset)
-        self.name = self.surf.name
+        # self.surf = SurfaceDataset(surf_path, offset)
+        type = 'white'
```

Comment:
`type` is a built-in function in python, an alternative variable name may be better